src/blocks/cpu.rs

```diff
@@ -239,3 +242,33 @@ impl Block for Cpu {
         &self.id
     }
 }
+
+#[inline]
+fn format_utilization(values: &[f64], count: usize, per_core: bool) -> String {
+    if per_core {
+        values
+            .iter()
+            .take(count)
+            .skip(1) // The first value is a global one.
+            .map(|v| format!("{:02.0}%", 100.0 * v))
```

Comment:
Is there a reason why only `per_core` appends the percentage sign?

Reply:
Same reason as above :slightly_smiling_face: